### ISSUE TITLE
Eik-cli doesn't work on Windows

### ISSUE THREAD

**Issue by rlf** [eik-lib/common#302]
It turns out the eik-cli relies on the glob module, which doesn't really work with the windows folder-separator.

See https://stackoverflow.com/questions/72313482/glob-paths-dont-work-in-my-windows-environment

Ref: https://github.com/eik-lib/common/blob/master/lib/helpers/resolve-files.js#L42

Reproduce:
```
try running `npm eik publish --debug` on any windows machine with a `files` pattern targetting any folder.
```

**Comment by wkillerud** [comment]
It'll take a bit more for the CLI package to run on Windows. This PR is for the CLI module:
https://github.com/eik-lib/cli/pull/584/files

**Comment by wkillerud** [comment]
This should be fixed in [v3.1.3](https://github.com/eik-lib/cli/releases/tag/v3.1.3) @rlf
